tests/sentry/incidents/test_tasks.py

```diff
@@ -190,3 +197,86 @@ def test(self):
                 handle_trigger_action.delay(self.action.id, incident.id, self.project.id, "fire")
             mock_handler.assert_called_once_with(self.action, incident, self.project)
             mock_handler.return_value.fire.assert_called_once_with()
+
+
+class ProcessPendingIncidentSnapshots(TestCase):
+    @pytest.mark.pending
```

Comment:
Probably should remove? Btw, if you want to run a specific class, you can use 
`py.test tests/sentry/incidents/test_tasks.py::ProcessPendingIncidentSnapshots`